src/main/resources/openapi/event.yaml

```diff
@@ -371,15 +440,22 @@ components:
 
     EventMapping:
       type: object
+      required:
+        - type
+        - name
+        - has_restrictions
       properties:
         id:
           type: integer
         type:
           type: string
+          maxLength: 256
         sub_type:
           type: string
+          maxLength: 256
         name:
           type: string
+          maxLength: 256
         handler:
           type: string
```

Comment:
Add constraints for `minLength` and `maxLength` here.

N.B the constraints only apply when a non-null value is provided, so it doesn't impact whether the field is required or not.